hooks/pre_commit/42-evr_extension_permissions.rb

```diff
@@ -0,0 +1,31 @@
+# Managed databases will be handled automatically.
+return if local_postgresql?
```

Comment:
```suggestion
return if local_postgresql?
return unless katello_enabled?
```

Otherwise this will run on Proxies and plain Foreman installs, which do not have the EVR extension, which will make the SQL below return no rows and run into the `else` branch, raising `fail_and_exit("Failed to check the ownership of the evr extension.")`